starkinfra/dynamicbrcode/__dynamicbrcode.py

```diff
@@ -0,0 +1,239 @@
+    - version [integer]: Integer that represents how many times the Brcode was updated.
+    - created [datetime.date or string]: Creation datetime of the DynamicBrcode. ex: datetime.datetime(2022, 5, 17)
+    - key_id [string]: Receiver's Pix Key id. Can be a tax_id (CPF/CNPJ), a phone number, an email or an alphanumeric sequence (EVP). ex: "+5511989898989"
+    - status [string]: Brcode status. Options: "created", "overdue", "paid", "voided", "canceled" or "expired"
+    - reconciliation_id [string]: Id to be used for conciliation of the resulting Pix transaction. ex: "b77f5236-7ab9-4487-9f95-66ee6eaf1781"
+    - amount [integer]: A positive integer that represents the amount in cents to be transferred. ex: 100 (R$1.00).
+    ## Parameters (conditionally-required):
+    - cashier_type [string]: Cashier's type. Required if the cash_amount is different than 0. Options: "merchant", "participant" and "other"
+    - cashier_bank_code [string]: Cashier's bank code. Required if the cash_amount is different than 0. ex: "20018183"
+    ## Parameters (optional):
+    - cash_amount [integer]: Amount to be withdrawal from the cashier in cents. ex: 1000 (= R$ 10.00)
+    - reason [string]: denial reason. Options: "invalidAccountNumber", "blockedAccount", "accountClosed", "invalidAccountType", "invalidTransactionType", "taxIdMismatch", "invalidTaxId", "orderRejected", "reversalTimeExpired", "settlementFailed"
+    - expiration [integer]: Time in seconds counted from the creation datetime until the DynamicBrcode expires. After expiration, the Brcode cannot be paid anymore. Default value: 86400 (1 day)
+    - sender_name [string]: Sender's full name. ex: "Anthony Edward Stark"
+    - sender_tax_id [string]: Sender's tax_id (CPF/CNPJ). ex: "01.001.001/0001-01"
+    - description [string]: Additional information to be delivered to the receiver.
+    - amount_type [string]: Amount type of the Brcode. If the amount type is fixed the Brcode ammount cannot be changed after creation. Options: "fixed"or "custom"
+    ## Return:
+    - Dumped JSON string that must be returned to us
+    """
+    return dumps({
+        "version": version,
+        "created": created,
+        "keyId": key_id,
+        "status": status,
+        "reconciliationId": reconciliation_id,
+        "amount": amount,
+        "reason": reason or "",
+        "expiration": expiration or 0,
+        "senderName": sender_name or "",
+        "senderTaxId": sender_tax_id or "",
+        "description": description or "",
+        "amountType": amount_type or "",
+        "cashAmount": cash_amount or 0,
+        "cashierType": cashier_type or "",
+        "cashierBankCode": cashier_bank_code or "",
+    })
+
+
+def verify(uuid, signature, user=None):
+    """# Verify a DynamicBrcode Read 
+    When an DynamicBrcode is read by your user, a GET request will be made to the URL stored in the Brcode to 
+    retrieve additional information needed to complete the transaction.
```

Comment:
When a DynamicBrcode is read by your user, a GET request containing the Brcode's UUID will be made to your registered URL to retrieve additional information needed to complete the transaction.